### PR TITLE
Add test for `policy-scc.yaml`

### PR DESCRIPTION
**Pull request by dhaiducek**
Notes:
- Reduces package-level variables to prevent collisions and allow for similarly named variables in each test file
- Adds test for `policy-scc.yaml`
  - Along with the package-level variable reduction, with this test file I attempted to establish a boilerplate test format where a new test for a new kind could be written with minimal updates (primarily to the variable declarations at the top) and then built upon

Addresses:
- https://github.com/stolostron/backlog/issues/20846

### CODE REVIEW

**Review by ycao56 — APPROVED** [review]
This is great!

File: test/integration/policy_scc_test.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+		By("Patching placement rule")
+		err = common.PatchPlacementRule(
+			userNamespace, "placement-"+rootPolicyName, clusterNamespace, kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		By("Checking that " + rootPolicyName + " exists on the Hub cluster")
+		rootPolicy := utils.GetWithTimeout(
+			clientHubDynamic, common.GvrPolicy, rootPolicyName, userNamespace, true, defaultTimeoutSeconds,
```
Comment (ycao56) [inline test/integration/policy_scc_test.go:64]: policy name is `policy-securitycontextconstraints`

Comment (ycao56) [reply test/integration/policy_scc_test.go:64]: file name is `policy-scc`

Comment (dhaiducek) [reply test/integration/policy_scc_test.go:64]: Oh, man! Fixing now. Looks like SCC isn't namespaced, either. And this particular one might be preinstalled on clusters, so it's probably always compliant...

**Review by ycao56 — APPROVED** [review]

**Comment by dhaiducek** [comment]
@ycao56 I just noticed that the label on `policy-collection-test` was updated so it would work as intended. Should I add it back or leave it removed?

**Comment by ycao56** [comment]
> @ycao56 I just noticed that the label on the `policy-collection-test` Make target was updated so it would work as intended. Should I add it back or leave it removed?

It's fine to be removed.

**Pull request merged**